sites/skeleton.dev/src/content/docs/components/toast/react.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+---
+
+export const components = componentSet;
+
+import Example from '@examples/components/toast/Example.tsx';
+import ExampleRaw from '@examples/components/toast/Example.tsx?raw';
+import ExampleType from '@examples/components/toast/ExampleType.tsx';
+import ExampleTypeRaw from '@examples/components/toast/ExampleType.tsx?raw';
+import ExamplePlacement from '@examples/components/toast/ExamplePlacement.tsx';
+import ExamplePlacementRaw from '@examples/components/toast/ExamplePlacement.tsx?raw';
+import ExamplePromise from '@examples/components/toast/ExamplePromise.tsx';
+import ExamplePromiseRaw from '@examples/components/toast/ExamplePromise.tsx?raw';
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<Example client:visible />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={ExampleRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+## Type
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<ExampleType client:visible />
```

Comment:
For the buttons, I recommend using a grid layout with at `4x1` by default and `2x2` or smaller screens so it doesn't get squished on small mobile devices.

![Screenshot 2025-03-24 at 1 09 13 PM](https://github.com/user-attachments/assets/5aef88a2-9a7c-4112-9bdf-db95ab23f2dc)